README.md

```diff
@@ -10,7 +10,16 @@ You can install the package via composer:
 composer require bfabio/publiccode-parser-php
 ```
 
-It only supports Linux x86_64 for now.
+### Supported Platforms
+
+This library supports the following platforms:
+
+- **Linux x86-64** (AMD64) - Fully supported
+- **Linux ARM64** (aarch64) - Fully supported
+- **macOS Intel** (x86-64) - Fully supported
```

Comment:
Aren't Intel Macs pretty much on their way out? We can probably skip the support for those